src/trading/universal_trader.py

```
@@ -649,68 +514,83 @@ async def _monitor_position_until_exit(
                 else:
                     # Log current status
                     pnl = position.get_pnl(current_price)
-                    logger.debug(
-                        f"Position status: {current_price:.8f} SOL ({pnl['price_change_pct']:+.2f}%)"
-                    )
+                    logger.debug(f"Position status: {current_price:.8f} SOL ({pnl['price_change_pct']:+.2f}%)")
 
                 # Wait before next price check
                 await asyncio.sleep(self.price_check_interval)
 
             except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)